src/Http/Http.Results/src/TypedResults.cs

```diff
@@ -471,13 +471,15 @@ public static JsonHttpResult<TValue> Json<TValue>(TValue? data, JsonSerializerCo
     /// Used to configure the <c>Last-Modified</c> response header and perform conditional range requests.</param>
     /// <param name="entityTag">The <see cref="EntityTagHeaderValue"/> to be configure the <c>ETag</c> response header
     /// and perform conditional requests.</param>
+    /// <param name="enableRangeProcessing">Set to <c>true</c> to enable range requests processing.</param>
     /// <returns>The created <see cref="PushStreamHttpResult"/> for the response.</returns>
     public static PushStreamHttpResult Stream(
         Func<Stream, Task> streamWriterCallback,
         string? contentType = null,
         string? fileDownloadName = null,
         DateTimeOffset? lastModified = null,
-        EntityTagHeaderValue? entityTag = null)
+        EntityTagHeaderValue? entityTag = null,
+        bool enableRangeProcessing = false)
```

Comment:
FYI we avoid binary breaking changes like this, existing APIs cannot be modified, even if it's source compatible, this would have to be a whole new method.